docs/en/oas3/OAS3-PDND-WP.yaml

```diff
@@ -0,0 +1,299 @@
+            DPoP proof JWT, to comply with the REST_JWS_2021_POP security
+            pattern using the POP_DPoP implementation. See also <a target="blank"
+            href="https://datatracker.ietf.org/doc/html/rfc9449.html">RFC
+            9449</a>.<br/><br/>
+
+
+            <a target="blank" href="https://jwt.io/#debugger-io?token=eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwia2V5X29wcyI6WyJzaWduIl0sImtpZCI6IjM5ZmE5NjBiLTc3M2YtNDllZi04YTBlLWU3NzNlOWI5N2FlOCIsImNydiI6IlAtMjU2IiwieCI6Imh1eVhJUU52OTAyb0xzcFg0X3pvbkM5NEc2eUVsbjZsc2RtLTF3TTczMm8iLCJ5IjoiSTlQREVhd1dIcWFGREd4MVprTmstMlBWNldkcGNhSDNBZk9iQlNMaWhndyJ9fQ.eyJqdGkiOiIyYzc2ZmNhMy1jYjRlLTQzMTItOGI2ZS05NzQ5NDYyZjQyMGQiLCJodG0iOiJQT1NUIiwiYXRoIjoiNDc1MmMzMmQ2YzQ4NzYzZjBmMzljZDNkYzk5ZDJlOTk3OTMyYmFmMzc1NjNiYzVhODk5NDg3YTZmODZlNWIxZCIsImh0dSI6Imh0dHBzOi8vd2FsbGV0LXByb3ZpZGVyLmV4YW1wbGUuaXQiLCJpYXQiOjE3NjIyNjI2MTZ9.SQPvfKbiXVZU3Wcyz1ukBguYKA37XKfSUEM5WjdshvjeaZM5lv4r7_HP1xkYcmazpHPh_mxdAmjeNoDbr4N_JQ">EXAMPLE
+            ON JWT.IO</a>
+          required: false
+          schema:
+            type: string
+            format: JWT
+            example: >-
+              eyJ0eXAiOiJkcG9wK2p3dCIsImFsZyI6IkVTMjU2IiwiandrIjp7Imt0eSI6IkVDIiwia2V5X29wcyI6WyJzaWduIl0sImtpZCI6IjM5ZmE5NjBiLTc3M2YtNDllZi04YTBlLWU3NzNlOWI5N2FlOCIsImNydiI6IlAtMjU2IiwieCI6Imh1eVhJUU52OTAyb0xzcFg0X3pvbkM5NEc2eUVsbjZsc2RtLTF3TTczMm8iLCJ5IjoiSTlQREVhd1dIcWFGREd4MVprTmstMlBWNldkcGNhSDNBZk9iQlNMaWhndyJ9fQ.eyJqdGkiOiIyYzc2ZmNhMy1jYjRlLTQzMTItOGI2ZS05NzQ5NDYyZjQyMGQiLCJodG0iOiJQT1NUIiwiYXRoIjoiNDc1MmMzMmQ2YzQ4NzYzZjBmMzljZDNkYzk5ZDJlOTk3OTMyYmFmMzc1NjNiYzVhODk5NDg3YTZmODZlNWIxZCIsImh0dSI6Imh0dHBzOi8vd2FsbGV0LXByb3ZpZGVyLmV4YW1wbGUuaXQiLCJpYXQiOjE3NjIyNjI2MTZ9.SQPvfKbiXVZU3Wcyz1ukBguYKA37XKfSUEM5WjdshvjeaZM5lv4r7_HP1xkYcmazpHPh_mxdAmjeNoDbr4N_JQ
+        - name: Agid-JWT-Signature
+          in: header
+          description: >-
+            JWT containing the signature of the message headers whose integrity
+            needs to be guaranteed, to comply with the INTEGRITY_REST_02
+            security pattern (see <a target="blank"
+            href="https://italia.github.io/eid-wallet-it-docs/v1.0.0/en/e-service-pdnd.html">e-Service PDND</a>). <br/><br/>
+
+
+            <a target="blank" href="https://jwt.io/#debugger-io?token=eyJhbGciOiJFUzI1NiIsImtpZCI6ImQ0YzNiMmExLTk4NzYtNTQzMi0xMGZlLWRjYmE5ODc2NTQzMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiI4MjkxNGIzZi02MGIyLTQ1MjktYjRkNi0zZDRlNjdmMGE5MzMiLCJzdWIiOiI4MjkxNGIzZi02MGIyLTQ1MjktYjRkNi0zZDRlNjdmMGE5MzMiLCJhdWQiOiJodHRwczovL3dhbGxldC1wcm92aWRlci5leGFtcGxlLml0IiwiaWF0IjoxNzMzMzk3ODQwLCJuYmYiOjE3MzM0MDE2MjgsImV4cCI6MTczMzQwMTQ0MCwianRpIjoiZDNmN2IyYzktMjc0YS00MmI3LThmOGQtMmU5ZDhiMTczNGIwIiwic2lnbmVkX2hlYWRlcnMiOlt7ImRpZ2VzdCI6IlNIQS0yNTY9NzJlMThiZGRkZjEzYzkxMWI0ZGQ1NjJlZTIxOTc5YTVjOWYyMzVjM2EwMWJkMTQyNmU4NTdkOGMxYTI4MmY0MSJ9LHsiY29udGVudC10eXBlIjoiYXBwbGljYXRpb24vanNvbiJ9XX0.oKBWAjKB194EWGm5jZDVGT4ravReUQSmsN_yAGQL-M7bGr2hcrOBSMjj2cxhd0RVluasMunZqHz2eIss6EO1wQ">EXAMPLE
```

Comment:
we should replace in the future https://jwt.io with a tool deployed in github italia.

please move this into a new isse :-)